docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,81 @@
+# System Security Plan (SSP) Information
+
+Systems that fall under the [CU Data Classification](https://www.cu.edu/data-governance/resources-support/data-classification) of **Confidential Information** or
+**Highly Confidential Information** are required to provide a System Security Plan (SSP).
+An SSP is a document that outlines how an organization protects the security of a system.
+An SSP is a critical document that helps organizations meet compliance requirements.
+
+The [OIT IT Security](https://oit.colorado.edu/services/it-security) group is responsible for guiding, evaluating and approving your SSP.
+[OIT IT Security](https://oit.colorado.edu/services/it-security) prescribes the [NIST SP 800-171 Rev. 2](https://csrc.nist.gov/pubs/sp/800/171/r2/upd1/final)
+control set as the official standard to meet compliance requirements for **Confidential Information** and **Highly Confidential Information**.
+
+Getting your SSP completed and approved is not only necessary, but can be a rigorous process.
+The good news is that by using the OIT Research Computing [Cloud Foundations Service](https://www.colorado.edu/rc/cloudfoundations) (CFS)
+**LCA1** solution, some of your security burden is handled by our service.
+
+The CFS **LCA1** solution is an [AWS Landing Zone](https://docs.aws.amazon.com/prescriptive-guidance/latest/migration-aws-environment/understanding-landing-zones.html)
+that is certified to support both **Confidential Information** and **Highly Confidential Information**.
```

Comment:
Do we really want to make the statement about Highly Confidential?  I wasn't sure how we want to state this.

Reply:
I updated the LCA1 support indicating that highly confidential is on the horizon.